src/api/authorData.js

```diff
@@ -31,10 +37,14 @@ const getSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 });
 
 // DONE: FILTER FAVORITE AUTHORS
-const favAuthors = () => new Promise((resolve, reject) => {
-  axios.get(`${dbUrl}/authors.json?orderBy="favorite"&equalTo=true`)
-    .then((response) => resolve(Object.values(response.data)))
-    .catch((error) => reject(error));
+const favAuthors = (uid) => new Promise((resolve, reject) => {
+  // axios.get(`${dbUrl}/authors.json?orderBy="favorite"&equalTo=true`)
```

Comment:
Be sure we are not pushing and merging commented-out code. If you need to save it you should add it to the readme.